public/js/components/Info.js

```diff
@@ -0,0 +1,154 @@
+const React = require("react");
+const { connect } = require("react-redux");
+const classnames = require("classnames");
+const { getTabs } = require("../selectors");
+
+require("./Info.css");
+const { DOM: dom } = React;
+const ImPropTypes = require("react-immutable-proptypes");
+
+const githubUrl = "https://github.com/devtools-html/debugger.html/blob/master";
+
+function getTabsByBrowser(tabs, browser) {
+  return tabs.valueSeq()
+    .filter(tab => tab.get("browser") == browser);
+}
+
+const Info = React.createClass({
+  propTypes: {
+    tabs: ImPropTypes.map.isRequired
+  },
+
+  displayName: "Info",
+
+  getInitialState() {
+    return {
+      selectedPane: "Firefox"
+    };
+  },
+
+  renderTabs(tabTitle, tabs, paramName) {
+    if (!tabs || tabs.count() == 0) {
+      return null;
```

Comment:
I'd like to, the issue I run into is flashing a message that is replaced as soon as the connection is established. It would be nice if we tracked whether a browser was connected, that way the message could read "Chrome is not connected" or something like that.